subsys/bluetooth/controller/ll_sw/nordic/lll/lll_conn_iso.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2018-2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
The copyright is a bit fun, since this is technically added by Oticon/Demant rather than Nordic, but it makes more sense to have Nordic here in the Nordic LLL. 

Perhaps the year should just be 2021 though?

Reply:
I agree it's a bit odd. I'll put in whatever you like ;-)

Reply:
I think it makes the most sense to just have Nordic there, but please update it to just 2021 to better reflect when it was added.